superset-frontend/src/cccs-viz/plugins/plugin-chart-at-a-glance-user-id/README.md

```diff
@@ -0,0 +1,53 @@
+## @superset-ui/plugin-chart-at-a-glance
+
+
+
+This plugin provides At A Glance for Superset.
+
+### Usage
+
+Configure `key`, which can be any `string`, and register the plugin. This `key` will be used to lookup this chart throughout the app.
+
+```js
+import AtAGlanceChartPlugin from '@superset-ui/plugin-chart-at-a-glance';
+new AtAGlanceChartPlugin()
+  .configure({ key: 'at-a-glance' })
```

Comment:
This probably doesn't matter, but the other keys seem to use underscores. Also, should this be at_a_glance_user_id (or something to make it more specific, like the others)?

Reply:
it should be underscore, will change

Reply:
actually this README is a copy and paste from the IP at a glance and is not even correct.  I don't think I ever modified this file, I'll see if I can make it more correct.